config.ru

```diff
@@ -1,11 +1,21 @@
 require 'sinatra'
 require 'sinatra/reloader' if development?
+require 'open3'
+
+helpers do
```

Comment:
Do you need "helpers" block here? I thought helpers made this method available in the view, but all you are doing is using the method directly. I think you can get by with just `def parse_ps(ps_string)`.